packages/rs-drive-abci/src/execution/check_tx/v0/mod.rs

```diff
@@ -113,6 +113,8 @@ where
 
         let unique_identifiers = state_transition.unique_identifiers();
 
+        let priority = state_transition.fee_multiplier();
```

Comment:
If we use only multiplier as a priority then simple document transition with cheap processing fee with multiplier 3 will be committed faster than identity creation with processing fees higher than the multiplied document transition fees. We probably should put here multiplied fees of ST so the most expensive will go first. You probably want to use somehow nonce in this number (if it's possible) as well to sort txs by nonce so we reject less txs due to nonce is to far in future.

Reply:
I think you are missing the point... We don't want the thing with the most fees to go first. We want the thing with the highest multiplier to go first. Remember a more expensive state transition might take more time or effort. The fee multiplier is based on effort.

So we basically want the highest cost per effort to go first.

Reply:
Ok, you implementing different feature when I was thinking about.